### DISCUSSION TITLE
Prevent clicks in CoreWindow.PointerPressed

### DISCUSSION THREAD

**Discussion by HakanL** [unoplatform/uno#10695] [Q&A]
I have a use case where I need to prevent clicks in certain cases (basically I'm turning off the screen on Linux, but the touch screen is still "active", I want to turn on the screen before accepting the clicks).
I have `CoreWindow.GetForCurrentThread().PointerPressed` hooked up, which I use to turn on the display, and it works great. But in the event arguments there is a property called `Handled`, which following normal Windows events, if you set it to true, that means lower event handlers ignore the event. So I'm setting this to true to prevent the clicks from reaching the views. But it doesn't work, it doesn't seem like it's checked (or wiring up the PointerPressed event when creating the FrameBufferHost isn't early enough. Is there a way to solve this?

**Comment by jeromelaban** [comment]
@dr1rrb may be able to help

**Reply by dr1rrb** [reply]
Humm, if I understand correctly you are subscribing to the `CoreWindow.PointerPressed` event and you are setting the `PointerEventArgs.Handled = true;` but this does not prevent bubbling of the pointer event. 

Then indeed we are not validating the `Handled` flag when we raise handlers here https://github.com/unoplatform/uno/blob/master/src/Uno.UWP/UI/Core/CoreWindow.Pointers.Managed.cs#L57, but even if we fix that, we would have to change the wiring of the `InputManager` here https://github.com/unoplatform/uno/blob/master/src/Uno.UI/UI/Xaml/Internal/InputManager.Pointers.Managed.cs#L79 to be the **last** handler invoked.

Are you able to inject a temp layer on top of the app when the screen goes off? Then if you handled the event on it, it will prevent bubbling of the event (we are properly validating the `Handled` on the _routed_  event args).

BTW, the `CoreWindow.PointerPressed` is implemented only on skia.

**Comment by HakanL** [comment]
Yes, that is correct, I'm subscribing to that event and setting Handled to true. I guess I could add a layer, but it feels a little too much of a bandaid solution, any other options? This is only needed on Skia (Linux).

**Reply by dr1rrb** [reply]
Yes it's definitely a patch ... but so far I don't see any other option that would prevent bubbling of the first event except maybe open a `Popup` with light dismiss enabled, but basically it will do the same.
